plugins/GSdx/Renderers/DXCommon/GSRendererDX.cpp

```diff
@@ -354,6 +357,15 @@ void GSRendererDX::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sourc
 	ASSERT(m_dev != NULL);
 	dev = (GSDeviceDX*)m_dev;
 
+	bool hdr_colclip = m_env.COLCLAMP.CLAMP == 0;
+	if (hdr_colclip)
+	{
+		GSVector4 sRect(0, 0, 1, 1);
+		GSVector4 dRect(0, 0, rtsize.x, rtsize.y);
+		hdr_rt = dev->CreateTexture(rtsize.x, rtsize.y, DXGI_FORMAT_R16G16B16A16_FLOAT);
+		dev->StretchRect(rt, sRect, hdr_rt, dRect, ShaderConvert_COPY, false);
```

Comment:
After this line put the strechrect mod256 from hdr_rt to tmp.

Then add a save call with drawcall id as parameter. Look at GSRendererHw.cpp for an example